learn.qmd

```diff
@@ -1,13 +1,82 @@
+| [{{< fa regular window-maximize >}} Reading and cleaning data for outbreak analytics with R](https://epiverse-trace.github.io/tutorials-early/)           | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/tutorials-early)  | [{{< fa solid person-chalkboard >}} View](https://epiverse-trace.github.io/tutorials-early/instructor/index.html)  | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |
+| [{{< fa regular window-maximize >}} Real-time analysis and forecasting for outbreak analytics with R](https://epiverse-trace.github.io/tutorials-middle/) | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/tutorials-middle) | [{{< fa solid person-chalkboard >}} View](https://epiverse-trace.github.io/tutorials-middle/instructor/index.html) | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |
+| [{{< fa regular window-maximize >}} Scenario modelling for outbreak analytics with R](https://epiverse-trace.github.io/tutorials-late/)                   | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/tutorials-late)   | [{{< fa solid person-chalkboard >}} View](https://epiverse-trace.github.io/tutorials-late/instructor/index.html)   | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |
+
+### How-to guides in English
+
+| Material                                                                                                                  | Repository                                                               | Instructor schedule | Development stage                                                                                                                                      |
+| ------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [{{< fa regular window-maximize >}} How-to guides for Outbreak Analytics with R](https://epiverse-trace.github.io/howto/) | [{{< fa brands github >}} Repo](https://github.com/epiverse-trace/howto) | (empty)             | experimental, WIP [https://github.com/epiverse-trace/howto?tab=readme-ov-file#howto](https://github.com/epiverse-trace/howto?tab=readme-ov-file#howto) |
```

Comment:
could we replace this `(empty)` with empty content ` `? as in the [galaxy reference](https://training.galaxyproject.org/training-material/topics/covid19/)?